torch/_dynamo/variables/higher_order_ops.py

```diff
@@ -2170,14 +2171,40 @@ def call_function(
         )
 
 
-class AutoFunctionalizeHigherOrderVariable(TorchHigherOrderOperatorVariable):
+class PassthroughHigherOrderVariable(TorchHigherOrderOperatorVariable):
+    """
+    A Variable to proxy a call to the HOP into the Dynamo output graph.
+    Used by Compiled Autograd to let backward HOPs survive until AOTDispatcher.
+    """
+
+    def proxy_submod(self, tx, arg):
+        assert isinstance(arg.source, DictGetItemSource)
```

Comment:
This is assuming we're coming from compiled autograd tracing, which may not always be the case